checkout-ui-custom/src/partials/Deliver/AddAddressManualForm.js

```diff
@@ -0,0 +1,250 @@
+import { getBestRecipient } from './utils';
+
+export const ADD_ADDRESS_FORM_MANUAL_RECIEVER_PHONE_ID = 'bash--input-add-address-manual-form-receiverPhone';
+
+const Heading = () => /* html */ `
+    <div class="bash--heading">
+      <h3>Add a new delivery address</h3>
+      <a href="#" class="back-button--select" data-view="select-address">&lt; Back</a>
+    </div>
+`;
+
+const SaveButton = () => /* html */ `
+  <button
+    class="submit btn-go-to-payment btn btn-large btn-success"
+    id="btn-save-address"
+    type="submit">
+    Save Address
+  </button>
+`;
+
+const AddAddressManualForm = () => {
+  const fields = [
+    {
+      name: 'addressId',
+      type: 'hidden',
+      value: '',
+      required: false,
+    },
+    {
+      name: 'addressName',
+      type: 'hidden',
+      value: '',
+      required: false,
+      maxLength: 50,
+    },
+    {
+      name: 'lat',
+      required: false,
+      type: 'hidden',
+      value: '',
+    },
+    {
+      name: 'lng',
+      required: false,
+      type: 'hidden',
+      value: '',
+    },
+    {
+      name: 'addressType',
+      label: 'Address type',
+      required: true,
+      type: 'radio',
+      options: [
+        { value: 'residential', label: 'Residential', checked: true },
+        { value: 'business', label: 'Business' },
+      ],
+    },
+    {
+      name: 'businessName',
+      label: 'Business name',
+      required: false,
+      value: '',
+      maxLength: 100,
+    },
+    {
+      name: 'companyBuilding',
+      label: 'Building/Complex and number',
+      required: false,
+      value: '',
+      maxLength: 100,
+    },
+    {
+      name: 'street',
+      label: 'Street address',
+      required: true,
+      value: '',
+    },
+    {
+      name: 'neighborhood',
+      label: 'Suburb',
+      value: '',
+      maxLength: 750,
+      required: true,
+    },
+    {
+      name: 'city',
+      label: 'City',
+      required: true,
+      value: '',
+      maxLength: 750,
+    },
+    {
+      name: 'postalCode',
+      label: 'Postal code',
+      value: '',
+      type: 'tel',
+      minlength: 4,
+      maxLength: 4,
+      required: true,
+    },
+    {
+      type: 'note',
+      required: false,
+      name: 'suburb-postal-reminder',
+      value: 'Make sure to specify the correct Suburb and Postal code so we can easily find your address.',
+    },
+    {
+      name: 'state',
+      label: 'Province',
+      type: 'dropdown',
+      required: true,
+      options: [
+        { value: '', label: 'Select' },
+        { value: 'EC', label: 'Eastern Cape' },
+        { value: 'FS', label: 'Free State' },
+        { value: 'GP', label: 'Gauteng' },
+        { value: 'KZN', label: 'KwaZulu-Natal' },
+        { value: 'LP', label: 'Limpopo' },
+        { value: 'MP', label: 'Mpumalanga' },
+        { value: 'NC', label: 'Northern Cape' },
+        { value: 'NW', label: 'North West' },
+        { value: 'WC', label: 'Western Cape' },
+      ],
+    },
+    {
```

Comment:
if you look at the design: no need for a country display, the next field already captures 'ZAF'
![image](https://github.com/user-attachments/assets/78967fde-eca6-4ab4-b8e8-1365f06fe766)